django_socio_grpc/utils/servicer_register.py

```diff
@@ -114,11 +125,44 @@ def register(self, service, service_file_path=None):
 
             add_server(service_class.as_servicer(), self.server)
         except ModuleNotFoundError:
-            if self.override_pb2_grpc_file_path is not None:
-                logger.error(
-                    f"PB2 module {self.override_pb2_grpc_file_path} not found. Please generate proto before launching server"
-                )
-                return
             logger.error(
-                f"PB2 module {self.app_name}.{self.grpc_folder}.{self.app_name}_pb2_grpc not found. Please generate proto before launching server"
+                f"PB2 module {path} not found. Please generate proto before launching server"
+            )
+
+    def get_grpc_folder(self):
+        base_dir = Path(settings.BASE_DIR)
+
+        if self.to_root_grpc:
+            path = base_dir / Path(grpc_settings.ROOT_GRPC_FOLDER) / self.app_name
```

Comment:
this is an overiding of the "/" operator for Path class that replace the os.path.join methods ?

Reply:
Yes